app/views/cart/show.html.erb

```diff
@@ -1,14 +1,21 @@
 <h1>Cart</h1>
 
-<% cart.contents.each do |item| %>
-  <section id="cart-<%= item.id %>">
-    <img src= <%= item.image %>>
-    <p>Name: <%= item.name %></p>
-    <p>Sold By: <%= link_to item.merchant.name, "/merchants/#{item.merchant.id}" %></p>
-    <p>Price: <%= number_to_currency(item.price) %></p>
-    <p>Count: <%= cart.item_count(item.id) %></p>
-    <p>Subtotal: <%= number_to_currency(cart.subtotal(item.id)) %></p>
-  </section>
+<% if cart.contents.empty? %>
+  <p>Your cart is currently empty</p>
+<% else %>
+  <% cart.contents.each do |item| %>
+    <section id="cart-<%= item.id %>">
+      <img src= <%= item.image %>>
+      <p>Name: <%= item.name %></p>
+      <p>Sold By: <%= link_to item.merchant.name, "/merchants/#{item.merchant.id}" %></p>
+      <p>Price: <%= number_to_currency(item.price) %></p>
+      <p>Count: <%= cart.item_count(item.id) %></p>
+      <%= link_to '+', "/cart/add/#{item.id}", method: :patch unless cart.inventory_limit?(item.id)  %>
```

Comment:
Should we add this check `unless cart.inventory_limit?(item.id)` to the 'Add Item to Cart' link on the item show page? Otherwise a user could exceed the item's inventory by adding the same item over and over from the item's show page using that 'Add Item to Cart' link rather than the '+' link on the cart page.